auto_submit/lib/requests/check_pull_request.dart

```diff
@@ -91,16 +91,18 @@ class CheckPullRequest extends AuthenticatedRequestHandler {
   }
 
   Future<bool> _processMerge(PullRequest pullRequest) async {
-    // TODO(Kristin): Merge this PR. https://github.com/flutter/flutter/issues/100088
-    bool merged = 1 < 2;
-    if (merged) {
-      log.info(
-          'Merged the pull request ${pullRequest.number} in ${pullRequest.base!.repo!.slug().fullName} repository.');
+    final RepositorySlug slug = pullRequest.base!.repo!.slug();
+    final int number = pullRequest.number!;
+    final GithubService gitHub = await config.createGithubService(slug);
+    PullRequestMerge mergeResult = await gitHub.merge(slug, number);
+    if (mergeResult.merged!) {
+      log.info('Merged the pull request $number in ${slug.fullName} repository.');
+      return true;
     } else {
-      log.info('Failed to merge the pull request ${pullRequest.number}');
+      log.info('Failed to merge the pull request $number');
```

Comment:
Could we log the message why this PR is not merged. Also instead of log.info, we at least need to promote to `log.warming`.

Reply:
Sure. I changed it to log.warning and also log the message for the merge result.